.github/workflows/integration-test.yml

```diff
@@ -50,7 +50,7 @@ jobs:
         with:
           path: OpenSearch-Dashboards
           repository: opensearch-project/OpenSearch-Dashboards
-          ref: '1.x'
+          ref: '2.x'
```

Comment:
Does this branch exist?

I thought `2.0` were on `main`

Reply:
you are correct, updating